launch/base_hardware.launch

```diff
@@ -95,9 +95,16 @@
             <param name="baud" value="115200" />
         </node>
 
+        <node pkg="joy" type="joy_node" name="joy_node">
```

Comment:
Can't be launching this by default

Reply:
As discussed this can be launched by default since its a hardware component.